src/timesync/timesyncd.c

```diff
@@ -50,24 +53,26 @@ static int load_clock_timestamp(uid_t uid, gid_t gid) {
                 if (geteuid() == 0) {
                         /* Try to fix the access mode, so that we can still
                            touch the file after dropping priviliges */
-                        r = fchmod(fd, 0644);
-                        if (r < 0)
-                                return log_error_errno(errno, "Failed to change file access mode: %m");
-                        r = fchown(fd, uid, gid);
+                        r = fchmod_and_chown(fd, 0644, uid, gid);
                         if (r < 0)
-                                return log_error_errno(errno, "Failed to change file owner: %m");
+                                log_warning_errno(r, "Failed to chmod or chown %s, ignoring: %m", CLOCK_FILE);
```

Comment:
not that it matters, but you could also write this as:

```c
log_warning_errno(r, "Failed to chmod or chown " CLOCK_FILE ", ignoring: %m");
```

and it would be a tiny bit more efficient...